docs/api/webview-tag.md

```diff
@@ -663,22 +663,6 @@ Corresponds to the points in time when the spinner of the tab starts spinning.
 
 Corresponds to the points in time when the spinner of the tab stops spinning.
 
-### Event: 'did-get-response-details'
```

Comment:
This event should be deprecated before it can be removed.
@codebytere Is there a document for upcoming deprecations? Or what is the exact process?